reverse_proxy/app/proxy.py

```
@@ -7,27 +8,76 @@
+        self.client_info = client_info
+        self.start_time = time.time()
+        self.total_bytes = 0
+        self.last_score = 0.0
+
+    def get_priority_score(self): # 優先度指標 = bits/sec
+        duration = time.time() - self.start_time
+        if duration < 0.1:
+            duration = 0.1  # 0除算防止
+
+        bps = (self.total_bytes * 8) / duration  # ビット毎秒に変換
+        self.last_score = 1.0 / (bps + 1.0) # bpsが大きいほどスコアが小さくなる（優先度が高くなる）
```

Ruff (0.14.8)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)